newscoop/src/Newscoop/NewscoopBundle/Security/Http/Authentication/FrontendLogoutSuccessHandler.php

```diff
@@ -50,4 +51,15 @@ public function onLogoutSuccess(Request $request)
 
         return parent::onLogoutSuccess($request);
     }
+
+    private function setTargetUrl(Request $request)
```

Comment:
Why new function if it's note reused anywhere? it makes it harder to read as you need to find this function.